scripts/layouts_tool/layoutsfilehandler.h

```diff
@@ -0,0 +1,73 @@
+#ifndef LAYOUTSFILEHANDLER_H
+#define LAYOUTSFILEHANDLER_H
+
+#include <QFile>
+#include <QDir>
+#include "layout.h"
+
+typedef QList<QStringList> LayoutNamesData;
+typedef KeyboardLayoutPointer (*GetLayout_t)(std::string layoutName);
+
+class LayoutsFileHandler {
+public:
+    LayoutsFileHandler();
+    virtual ~LayoutsFileHandler();
+
+    void open(QString &layoutsPath, QList<Layout> &layouts);
+    void save(QFile &f, QList<Layout> &layouts);
+
+    static const QString INDENT;
```

Comment:
All static class members should look like this: s_indent;
In this case kIndent would work as well. 
https://www.mixxx.org/wiki/doku.php/coding_guidelines